src/auth/auth.controller.ts

```
@@ -0,0 +1,57 @@
+  async login(
+    @Body() request: LoginRequest,
+    @Session() session: Record<string, any>,
+    @Res() res: Response,
+  ) {
+    {
+      await this.authService.login(request);
+
+      this.setSessionCookie(res, session.id);
+
+      this.sendResponse(res, HttpStatus.CREATED);
+    }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)